src/components/Availability/AvailabilityList.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { fetchAvailability } from "../../actions/availability";
+
+class AvailabilityList extends Component {
+  componentDidMount() {
+    // console.log("params", this)
+    this.props.fetchAvailability();
+  }
+
+  render() {
+    if (!this.props.entity) return null;
+    return (
+      <div>
+        <h1>{this.props.user.username}, your availability are:</h1>
+        <div>
+          {this.props.entity.map(e => (
+            <ul key={e.id}>
+              <li>
+                From {e.startDate.slice(0, 10)} at
+                {e.startDate.slice(11, 16)}
+                <br></br>
+                To {e.endDate.slice(0, 10)} at
+                {e.endDate.slice(11, 16)}
```

Comment:
Check out Moment.js for manipulating and displaying time and dates, it's great! (https://momentjs.com/)

Reply:
like this for example
```
moment(e.startDate).format("ddd, hA");                       // "Sun, 3PM"
```
The whole table with the ddd, HH, etc meanings is here https://momentjs.com/docs/#/displaying/